content/es/docs/reference/glossary/container-runtime.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Container Runtime
+id: container-runtime
+date: 2019-06-05
+full_link: /es/docs/reference/generated/container-runtime
+short_description: >
+El _Container Runtime_, entorno de ejecución de un contenedor, es el software responsable de ejecutar contenedores.
+
+aka:
```

Comment:
al parecer esta linea no debe estar vacia

`10:24:50 PM: Error: Error building site: "/opt/build/repo/content/es/docs/reference/glossary/container-runtime.md:9:1": failed to unmarshal YAML: yaml: line 8: could not find expected ':'`

pero el [documento original](https://raw.githubusercontent.com/kubernetes/website/master/content/en/docs/reference/glossary/container-runtime.md) lo esta

Reply:
@javisabalete El problema es la linea anterior, que necesita una identanción debido al `>`.

Reply:
cierto :man_facepalming: